autogpt_platform/backend/backend/util/settings.py

```diff
@@ -136,8 +136,14 @@ class Config(UpdateTrackingModel["Config"], BaseSettings):
         description="The port for agent server API to run on",
     )
 
+    platform_base_url: str = Field(
+        default="",
+        description="Must be set so the application knows where it's hosted at. "
+        "This is necessary to make sure webhooks find their way.",
+    )
+
     frontend_base_url: str = Field(
-        default="http://localhost:3000",
+        default="",
```

Comment:
why drop the default here?

Reply:
`FRONTEND_BASE_URL=http://localhost:3000` only makes sense for local dev. In most other scenarios it breaks stuff.

Reply:
yeah but they should be setting those variables anyway outside of local dev

Reply:
That only applies to `PLATFORM_BASE_URL`.